.clang-tidy

```diff
@@ -7,7 +7,8 @@ CheckOptions:
   - { key: readability-identifier-naming.MethodCase,              value: camelBack }
   - { key: readability-identifier-naming.FunctionCase,            value: camelBack }
   - { key: readability-identifier-naming.NamespaceCase,           value: lower_case }
-  - { key: readability-identifier-naming.MemberPrefix,            value: m_ }
+  - { key: readability-identifier-naming.PrivateMemberPrefix,     value: m_ }
```

Comment:
To which clang-tidy version refers this change?

Reply:
@dkroenke This should be available in clang-tidy 10, see: https://releases.llvm.org/10.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/readability-identifier-naming.html

At the moment I would not yet enforce this clang-tidy file since needs to mature a little while longer. But I strongly suggest that we all integrate clang-tidy into our IDE so that this file can mature faster.